.gitignore

```diff
@@ -7,3 +7,7 @@ coverage
 couchdb/seed-data
 newrelic.js
 newrelic_agent.log
+<<<<<<< HEAD
+mysql-db/csv
```

Comment:
If you pushed the csv file (for example when it was empty) to your repo, and THEN added it to your .gitignore, it will continue to push the csv file to your repo. If this is the case, delete the csv file from your repo and it should prevent it from re-pushing from your local when you push to your repo master.

Reply:
Ah good tip, thanks!